src/components/PurchaseOrder/util.js

```diff
@@ -64,3 +64,7 @@ export const isReopenAvailableForOrder = (order = {}) => {
 
   return hasLineItemsToReceive && isWorkflowStatusNotPending(order);
 };
+
+export const ifDisabledToChangePaymentInfo = (order = {}) => {
```

Comment:
mb is?

Reply:
`isDisabledToChangePaymentInfo` - is a boolean var, `ifDisabledToChangePaymentInfo` - a function that can tell us if it's disabled or not